features/kernel/src/main/feature/feature.xml

```diff
@@ -114,13 +61,11 @@
     </feature>
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
-    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="6.0.6_1">
+    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
         <feature version="[6.0,6.1)">pax-http</feature>
```

Comment:
❓ The `pax-http` feature is blacklisted, right? So this line won't do anything?

Reply:
only version 6.0.6 to 6.0.10 but not 6.0.11